aten/src/ATen/mps/MPSDevice.mm

```diff
@@ -99,8 +99,9 @@ static inline MTLLanguageVersion getMetalLanguageVersion(const id<MTLDevice>& de
     sampleGridWithSourceTensor:coordinateTensor:layout:normalizeCoordinates:relativeCoordinates:alignCorners:paddingMode:samplingMode:constantValue:name:)] == YES;
   static bool _macos_13_2_plus = [mpsCD instancesRespondToSelector:@selector(convolution3DWithSourceTensor:weightsTensor:descriptor:name:)] == YES;
   static bool _macos_13_3_plus = NO;
-  if (@available(macOS 13.3, *))
+  if (@available(macOS 13.3, *)) {
```

Comment:
This would not work, as we've discussed many times already. Should this cherry-picks include selector check as well?

Reply:
Yes, this has been fixed.